src/bosh-director/lib/bosh/director/jobs/update_stemcell.rb

```diff
@@ -141,6 +151,20 @@ def update_steps
         steps += 1 if @stemcell_sha1 # also verify remote stemcell
         steps + cloud_factory.all_configured_clouds.count * 3 # check, upload and save for each cloud
       end
+
+      def is_supported?(cloud)
+        info = cloud[:cpi].info
+        unless @stemcell_formats.nil? || info["stemcell_formats"].nil?
```

Comment:
For readability, can you please switch this to use an `if` instead of an `unless`.